runtime-light/stdlib/crypto/crypto-functions.cpp

```diff
@@ -115,3 +110,192 @@ task_t<int64_t> f$openssl_verify(const string &data, const string &signature, co
 
   co_return 1;
 }
+
+static bool ichar_equals(char a, char b) noexcept {
```

Comment:
Let's put it into an anonymous namespace. Also, it's also acceptable to define it as lambda inside `parse_algorithm`.